packages/fast-components-react-msft/src/caption/caption.props.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { ICaptionClassNameContract, IManagedClasses  } from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum CaptionLevel {
+    _1 = 1,
+    _2 = 2
+}
+
+export enum CaptionTag {
+    p = "p",
+    span = "span",
+    caption = "caption",
+    figcaption = "figcaption"
+}
+
+export interface ICaptionHandledProps {
+    /**
+     * The caption content
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * The caption level
+     */
+    level?: CaptionLevel;
```

Comment:
You might add more descriptive comments to the level and tag props

Reply:
add more descriptions